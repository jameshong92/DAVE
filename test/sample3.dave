// test operations
int main() {
    int a = 83;
    int b = 69;
    int c = a + b;
    print(c);
    a++;
    b--;
    print(a);
    print(b);
    bool y = true;
    print(y);
    y = (1!=2);
    print(y);
}
