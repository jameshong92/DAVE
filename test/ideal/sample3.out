152
84
68
true
true
